docs/storage.md

```diff
@@ -103,3 +103,57 @@ Note that on the read path, Prometheus only fetches raw series data for a set of
+collectd_varnish_derive{host="myserver.fqdn.com",varnish="request_rate",dimension="MAIN.client_req"} 2.3021666667e+01 1582226100000
+collectd_load{host="myserver.fqdn.com",type="midterm"} 1.5500000000e-02 1582226100000
+collectd_varnish_derive{host="myserver.fqdn.com",varnish="request_rate",dimension="MAIN.cache_hit"} 3.8054166667e+01 1582226100000
+collectd_df_complex{host="myserver.fqdn.com",df="var-log",dimension="free"} 5.8093906125e+10 1582226100000
+collectd_varnish_derive{host="myserver.fqdn.com",varnish="request_rate",dimension="MAIN.s_pipe"} 0 1582226100000
+collectd_varnish_derive{host="myserver.fqdn.com",varnish="request_rate",dimension="MAIN.cache_hit"} 3.8054166667e+01 1582226100000
+collectd_load{host="myserver.fqdn.com",type="shortterm"} 1.1000000000e-02 1582226100000
+collectd_varnish_derive{host="myserver.fqdn.com",varnish="request_rate",dimension="MAIN.client_req"} 2.3021666667e+01 1582226100000
+collectd_load{host="myserver.fqdn.com",type="shortterm"} 1.1000000000e-02 1582226100000
+collectd_load{host="myserver.fqdn.com",type="shortterm"} 1.1000000000e-02 1582226100000
+collectd_load{host="myserver.fqdn.com",type="longterm"} 2.5500000000e-02 1582226100000
+collectd_varnish_derive{host="myserver.fqdn.com",varnish="request_rate",dimension="MAIN.s_pipe"} 0 1582226100000
+collectd_load{host="myserver.fqdn.com"type="longterm"} 2.5500000000e-02 1582226100000
+collectd_varnish_derive{host="myserver.fqdn.com",varnish="request_rate",dimension="MAIN.cache_hit"} 3.8054166667e+01 1582226100000
+collectd_load{host="myserver.fqdn.com",type="midterm"} 1.5500000000e-02 1582226100000
+# EOF
+```
+
+Note, `[number value]` can be mixed as normal or scientific number as per your preference. You are free to put custom labels on each metric, don't forget that "relabelling rules" defined in prometheus will not be applied on them! You should produce the final labels on your import file.
+
+This format is simple to produce, but not optimized or compressed, so it's normal if your data file is huge.
+
+Before starting the import process, please verify if there is an overlap between the data you are importing, and what already exists in TSDB.
+If there is an overlap, you will need to shut down TSDB, and restart it with `--storage.tsdb.allow-overlapping-blocks`, else you do not need to shut down TSDB.
+However, it is recommended you do so anyway.
+Do not forget the `--storage.tsdb.retention.time=X` flag, if you not want to lose any imported data points.
+
+An example import command is: `tsdb import rrd_exported_data.txt /var/lib/prometheus/ --max-samples-in-mem=10000`
+
+You can increase `max-sample-in-mem` to speed up the process, but the value 10000 seems a good balance.
+This tool will create all Prometheus blocks (see [On-disk layout][On-disk layout] above), in a temporary workspace. By default temp workspace is /tmp/ according to the $TMPDIR env var, so you can change it if you have disk space issues (`TMPDIR=/new/path tsdb import [...]`) !
```

Comment:
Why don't we create them in the actual storage dir, with tmp suffix? /tmp is usually not big.

Reply:
The reason I chose to go this way, is this can be viewed as an external 'modification' to the db, and as such, I did not want to place anything into the live db folder until the import is well and truly completed. Overriding the `TMPDIR` var to place it somewhere better is I think a good way to go.

Reply:
I need to agree here.

I would simplify configuration here. What about just following flow:

Create new blocks in given directory. It's up to user if this dir is tmp, Prometheus Dir or White House Google Drive. Does not matter. We just write blocks as `<ID>.tmp` and use `os.Rename` to avoid partial reads. Done.


What do you think? (: